xray/plot/facetgrid.py

```diff
@@ -381,7 +380,7 @@ def map(self, func, *args, **kwargs):
 
         for ax, namedict in zip(self.axes.flat, self.name_dicts.flat):
             if namedict is not None:
-                data = self.data[namedict]
+                data = self.data.loc[namedict]
```

Comment:
The `.map` method was actually broken, but the tests passed because all the facet grid axes were labeled with integers